rsconf/package_data/mongod/logrotate.conf.jinja

```diff
@@ -0,0 +1,10 @@
+{{ mongod.log_d }}/*.log
+/var/log/mongodb/mongod.log {
+    missingok
+    notifempty
+    postrotate
+        kill -SIGUSR1 $(cat {{ mongod.pid_file_path }}) &> /dev/null
```

Comment:
incorrect (funny enough github flagged this):
<img width="205" alt="Screen Shot 2022-10-17 at 11 00 47 AM" src="https://user-images.githubusercontent.com/5495179/196238710-8d5f93f0-376d-48f8-bf90-e403699d0a63.png">

Reply:
Can you elaborate on incorrect? I think what I wrote redirects stdout and stderr to /dev/null which I believe is what I want.

https://www.gnu.org/software/bash/manual/html_node/Redirections.html#:~:text=and%20standard%20error%3A-,%26%3Eword,-and

Reply:
Well, you learn something everyday: https://superuser.com/a/1650179

I'm not sure I buy the reasoning, e.g. if you don't space things right you get different results, which is probably a more common issue:

```sh
$ echo & > x
[1] 20361

[1]+  Done                    echo
$ echo > & x
bash: syntax error near unexpected token `&'
```

In the first case, you wrote a zero length file and the command executed in background. In the second, it failed.

Anyway, it's fine to keep it. I won't change my reasoning.

I don't know why github flagged the & other than it being confused as to the file type.